arbnode/batch_poster.go

```diff
@@ -52,8 +52,13 @@ import (
 )
 
 var (
-	batchPosterWalletBalance      = metrics.NewRegisteredGaugeFloat64("arb/batchposter/wallet/balanceether", nil)
-	batchPosterGasRefunderBalance = metrics.NewRegisteredGaugeFloat64("arb/batchposter/gasrefunder/balanceether", nil)
+	batchPosterWalletBalance          = metrics.NewRegisteredGaugeFloat64("arb/batchposter/wallet/balanceether", nil)
+	batchPosterGasRefunderBalance     = metrics.NewRegisteredGaugeFloat64("arb/batchposter/gasrefunder/balanceether", nil)
+	baseFeeGauge                      = metrics.NewRegisteredGauge("arb/batchposter/basefee", nil)
+	blobFeeGauge                      = metrics.NewRegisteredGauge("arb/batchposter/blobfee", nil)
+	blockGasUsedPerBlockGasLimitGauge = metrics.NewRegisteredGaugeFloat64("arb/batchposter/blockgasusedperblockgaslimit", nil)
+	blobGasUsedPerBlobGasLimitGauge   = metrics.NewRegisteredGaugeFloat64("arb/batchposter/blobgasusedperblobgaslimit", nil)
```

Comment:
Instead of posting percentage, how about separate metrics for `batchposter/blobgas/limit`, `batchposter/blobgas/used` then we can easily graph in grafana.

Reply:
agreed